src/NServiceBus.AcceptanceTests/Recoverability/When_non_transactional_send_message_is_moved_to_error_queue.cs

```diff
@@ -8,14 +8,12 @@
     using NServiceBus.Pipeline;
     using NUnit.Framework;
 
-    public class When_non_transactional_message_is_moved_to_error_queue : NServiceBusAcceptanceTest
+    public class When_non_transactional_send_message_is_moved_to_error_queue : NServiceBusAcceptanceTest
```

Comment:
Looks like @timbussmann's comment about the name of this test hasn't been addressed yet.